_posts/2016-05-20-drupalcon-nola-recap.md

```diff
@@ -0,0 +1,118 @@
+
+ <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Nice commute alongside horse and buggy to the convention center en route to <a href="https://twitter.com/DrupalConNA">@DrupalConNA</a> <a href="https://twitter.com/hashtag/NOLA?src=hash">#NOLA</a> <a href="https://twitter.com/hashtag/DrupalCon2016?src=hash">#DrupalCon2016</a> <a href="https://t.co/BvvIEJqoga">pic.twitter.com/BvvIEJqoga</a></p>&mdash; Savas Labs (@Savas_Labs) <a href="https://twitter.com/Savas_Labs/status/729739990524174336">May 9, 2016</a></blockquote>
+ <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+ <span class="caption">Rolling through the French Quarter on the way in!</span>
+
+ After arriving, I walked the .75 miles (no exaggeration... it's 1 mile long) **_inside_** the convention center to get to the business summit. Although a little disorganized due to a last-minute personnel change of who would be running the summit, it proved to be a very open forum where leaders in in the Drupal community spoke very openly, honestly, and encouragingly about how they run their Drupal businesses. It was refreshing how genuinely the open source philosophy pervaded the consciousness of the leaders who were on panels and giving presentations.
+
+ <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Gaining insight from <a href="https://twitter.com/hashtag/Drupal?src=hash">#Drupal</a> leaders <a href="https://twitter.com/mediacurrent">@mediacurrent</a> <a href="https://twitter.com/prometsource">@prometsource</a> <a href="https://twitter.com/lullabot">@lullabot</a> <a href="https://twitter.com/phase2">@phase2</a> <a href="https://twitter.com/DrupalConNA">@DrupalConNA</a> <a href="https://twitter.com/hashtag/NOLA?src=hash">#NOLA</a> <a href="https://twitter.com/hashtag/Drupalcon?src=hash">#Drupalcon</a> <a href="https://t.co/cqvytd7KG7">pic.twitter.com/cqvytd7KG7</a></p>&mdash; Savas Labs (@Savas_Labs) <a href="https://twitter.com/Savas_Labs/status/729739985109327873">May 9, 2016</a></blockquote>
+ <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+ In between panels and presentations, the rest of us divvied up into smaller groups brainstorming around key issues and presented our collective expertise back to the larger group. Topics ranged from content strategy, to marketing tactics, to how to handle conflict with large personalities, and everything in between. The breadth of the coverage ensured that nearly everyone got something out of the summit. My reading list grew by an order of magnitude from the summit. Now if I could only find the time...
+
+ After the summit, I joined my teammates [Lisa](team/lisa-ridley/) and [Kosta](team/kosta-harlan/) at the opening reception. This was the first opportunity to see former coworkers and members of the community we were  personally acquainted with before the conference.
```

Comment:
@chrisarusso - ahh, the Jam link works below `I must admit, since we have a [Jam](/team/kosta-harlan) of our own`. So I think you just need a leading `/` before `team` in both of these links.